easy/24.js

```diff
@@ -1,14 +1,21 @@
 function filterArray(arr) {
-	
+  let deleteString = _.remove(arr, function(x) {

```

Comment:
Plus for using lodash, you can remove this variable deleteString, use arrow function and shorthand without "return" and curly brackets